.github/workflows/ci.yaml

```diff
@@ -15,18 +15,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Set up JDK 20
+      - name: Set up JDK 17
         uses: actions/setup-java@v3
```

Comment:
Does this PR need to be rebased with main? Feels like I saw these config changes in the last PR too.

Reply:
hm yeah maybe. I'll look into this. Thanks for catching that.

Reply:
+1